modules/integration_azure-postgresql/detectors-postgresql.tf

```
@@ -33,7 +33,7 @@ resource "signalfx_detector" "cpu_usage" {
   tags                    = compact(concat(local.common_tags, local.tags, var.extra_tags))
 
   program_text = <<-EOF
-        base_filter = filter('resource_type', 'Microsoft.DBforPostgreSQL/servers') and filter('primary_aggregation_type', 'true')
+        base_filter = filter('resource_type', 'Microsoft.DB?orPostgreSQL/servers') and filter('primary_aggregation_type', 'true')
```

```suggestion
        base_filter = filter('resource_type', 'Microsoft.DBForPostgreSQL/servers', 'Microsoft.DBforPostgreSQL/servers') and filter('primary_aggregation_type', 'true')
```
More readable in my opinion